contracts/HybridCustody.cdc

```diff
@@ -732,7 +736,9 @@ pub contract HybridCustody {
             self.acctOwner = to
             self.relinquishedOwnership = false
 
-            emit OwnershipGranted(id: self.uuid, child: self.acct.address, owner: to)
+            emit OwnershipUpdated(id: self.uuid, child: self.acct.address, owner: to, active: false)
+            //TODO do we need both events
+            emit AccountUpdated(id: self.uuid, child: self.acct.address, parent: to, proxy: false, active: false)
```

Comment:
Whichever path we choose, we won't need a second event. An event will be emitted by the user when they add the published Capability to their manager with `addOwnedAccount()`